config/config-server/resources/cruise-config.xsd

```diff
@@ -808,7 +808,7 @@
     </xsd:element>
     <xsd:simpleType name="filePathType">
         <xsd:restriction base="xsd:string" xml:space="default">
-            <xsd:pattern value="([^. ].+[^. ])|([^. ][^. ])|([^. ])"/>
+            <xsd:pattern value="(([.]\/)?[.][^. ]+)|([^. ].+[^. ])|([^. ][^. ])|([^. ])"/>
```

Comment:
@xli I think this will need versioning — see [this commit](https://github.com/gocd/gocd/commit/6b26201e18a19b74d38a84d02533e062254f1dea) on how it's done.

Please also squash the PRs.